crates/biome_html_formatter/tests/specs/html/elements/preformatted.html

```
@@ -0,0 +1,20 @@
+<div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)